src/tests/ie_test_utils/common_test_utils/ngraph_test_utils.cpp

```diff
@@ -5,235 +5,31 @@
-        default:
-            FAIL() << "Comparator for " << precision << " precision isn't supported";
-    }
-}
-
-void Compare(const std::vector<std::pair<ngraph::element::Type, std::vector<std::uint8_t>>> &expectedOutputs,
-                    const std::vector<InferenceEngine::Blob::Ptr> &actualOutputs,
-                    float threshold, float abs_threshold) {
-    for (std::size_t outputIndex = 0; outputIndex < expectedOutputs.size(); ++outputIndex) {
-        const auto &expected = expectedOutputs[outputIndex];
-        const auto &actual = actualOutputs[outputIndex];
-        Compare(expected, actual, threshold, abs_threshold);
-    }
-}
-} // namespace
-
-void TransformationTestsF::accuracy_check(std::shared_ptr<ov::Model> ref_function,
-                                          std::shared_ptr<ov::Model> cur_function) {
+void TransformationTestsF::accuracy_check(const std::shared_ptr<ov::Model>& ref_function,
+                                          const std::shared_ptr<ov::Model>& cur_function) {
     try {
         if (ref_function->is_dynamic() || cur_function->is_dynamic()) {
             return;
         }
-        std::vector<std::vector<uint8_t>> input_data;
-        ngraph::element::TypeVector types;
-        for (auto param : ref_function->get_parameters()) {
-            types.push_back(param->get_element_type());
-
-            auto layout = InferenceEngine::Layout::ANY;
-            if (ov::is_scalar(param->get_shape())) {
-                layout = InferenceEngine::Layout::SCALAR;
-            }
-            InferenceEngine::TensorDesc td(InferenceEngine::Precision::FP32, param->get_shape(), layout);
-            const auto &input = FuncTestUtils::createAndFillBlob(td);
-            const auto &input_size = input->byteSize();
-
-            std::vector<uint8_t> data;
-            data.resize(input_size);
-
-            auto memory = InferenceEngine::as<InferenceEngine::MemoryBlob>(input);
-            IE_ASSERT(memory);
-
-            const auto lockedMemory = memory->wmap();
-            const auto buffer = lockedMemory.as<const std::uint8_t *>();
-            std::copy(buffer, buffer + input_size, data.data());
-
-            input_data.push_back(std::move(data));
+        std::map<std::shared_ptr<ov::Node>, ov::Tensor> input_data;
+        for (const auto& param : ref_function->get_parameters()) {
+            const auto &tensor = ov::test::utils::create_and_fill_tensor(param->get_element_type(),
+                                                                        param->get_shape());
+            input_data[param] = tensor;
         }
 
-        auto ref_outputs = ngraph::helpers::interpreterFunction(ref_function, input_data, types);
-        auto outputs = ngraph::helpers::interpreterFunction(cur_function, input_data, types);
+        auto ref_outputs = ngraph::helpers::interpretFunction(ref_function, input_data);
+        auto outputs = ngraph::helpers::interpretFunction(ref_function, input_data);
```

Comment:
Looks like we compare inference for two the same functions here.

Reply:
Also I think input data must be copied for cur_function

Reply:
my mistake, corrected